Tests/SwiftFormatTests/Rules/UseLetInEveryBoundCaseVariableTests.swift

```diff
@@ -38,23 +54,45 @@ final class UseLetInEveryBoundCaseVariableTests: LintOrFormatRuleTestCase {
           "5️⃣",
           message: "move this 'let' keyword inside the 'case' pattern, before each of the bound variables"
         ),
+        FindingSpec(
+          "6️⃣",
+          message: "move this 'var' keyword inside the 'case' pattern, before each of the bound variables"
+        ),
+        FindingSpec(
+          "7️⃣",
+          message: "move this 'var' keyword inside the 'case' pattern, before each of the bound variables"
+        ),
       ]
     )
   }
 
   func testIfCase() {
-    assertLint(
+    assertFormatting(
       UseLetInEveryBoundCaseVariable.self,
-      """
-      if case 1️⃣let .labeled(label, value) = DataPoint.labeled("hello", 100) {}
-      if case .labeled(label, let value) = DataPoint.labeled("hello", 100) {}
-      if case .labeled(let label, let value) = DataPoint.labeled("hello", 100) {}
-      if case 2️⃣let .labeled(label, value)? = DataPoint.labeled("hello", 100) {}
-      if case 3️⃣let .labeled(label, value)! = DataPoint.labeled("hello", 100) {}
-      if case 4️⃣let .labeled(label, value)?? = DataPoint.labeled("hello", 100) {}
-      if case 5️⃣let (label, value) = DataPoint.labeled("hello", 100) {}
-      if case let x as SomeType = someValue {}
-      """,
+      input: """
+        if case 1️⃣let .labeled(label, value) = DataPoint.labeled("hello", 100) {}
```

Comment:
thoughts on adding a test for `case .labeled(let label, var value)` we should avoid flagging this pattern

Reply:
I think the existing tests (the next two lines after this) cover that pattern, where binding keywords are already used adjacent to the bound variables.